sev/.travis.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Red Hat
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+language: rust
+rust:
+  - stable
+  - beta
+  - nightly
+matrix:
+  allow_failures:
+    - rust: nightly
+  fast_finish: true
+install:
+  - rustup component add rustfmt
+  - cargo install --force cargo-audit
+  - cargo generate-lockfile
+script:
+  - cargo fmt -- --check
+  - cargo audit
+  - cargo test --verbose
+  - cargo test --verbose --features=openssl
```

Comment:
Enarx already has a Travis config, and this one looks mostly the same. Do we plan on amending this to complement the main config, or will the main config replace this one entirely?